source/index.jsx

```diff
@@ -44,11 +45,12 @@ render(
     <Router>
         <div>
             <Route exact path="/" component={Home}/>
-            <Route path="/sponsor" component={SponsorLetter} />
```

Comment:
Don't remove the sponsor route, that's where our sponsor tiers are located

Reply:
At the time, they were unused and empty, which was why I removed them.